airbyte-integrations/connectors/destination-bigquery/src/test/java/io/airbyte/integrations/destination/bigquery/typing_deduping/BigqueryDestinationHandlerTest.java

```diff
@@ -57,11 +56,10 @@ public void testToDialectType() {
   @Test
   public void testClusteringMatches() {
     StreamConfig stream = new StreamConfig(mock(),
-        SyncMode.INCREMENTAL,
         DestinationSyncMode.APPEND_DEDUP,
         List.of(new ColumnId("foo", "bar", "fizz")),
         Optional.empty(),
-        new LinkedHashMap<>());
+        new LinkedHashMap<>(), 0, 0, 0);
```

Comment:
.... do you want me to set default values for the generation/sync ID args? These diffs seem kind of dumb